backend/modules/query_controllers/default/controller.py

```diff
@@ -82,7 +82,10 @@ async def answer(self, request: DefaultQueryInput):
 
             # Get the answer
             logger.info(f"Request query: {request.query}")
-            outputs = qa({"query": request.query})
+
+            # Can't use ainvoke here as Qdrant uses grpc for the retrieval, 
+            # which terminates as the client grpc is set to false
```

Comment:
not sure, but for me this worked

Reply:
It is  due to using Qdrant from langchain as opposed to from langchain_community.